UnityProject/Assets/Scripts/Input System/MouseInputController.cs

```diff
@@ -473,8 +482,24 @@ private MouseDraggable GetDraggable()
 		return null;
 	}
 
-
-	private bool CheckAltClick()
+    /// <summary>
+    /// Fires if shift is pressed on click, initiates examine. Assumes inanimate object, but upgrades to checking health if living, and id if target has 
+	/// storage and an ID card in-slot.
+    /// </summary>
+    /// <returns>false if no shiftclick, true if something was examined (for future chatbox related events, or gui)</returns>
+    private bool CheckShiftClick()
```

Comment:
does this function `return true;` anywhere?

Reply:
changed to void